packages/nimble-components/src/rich-text/viewer/styles.ts

```diff
@@ -41,4 +45,20 @@ export const styles = css`
     li > p:empty {
         display: none;
     }
+
+    ${
+        /**
+         * When an absolute link is not HTTPS/HTTP, the anchor tag renders without an `href`, appearing as plain text.
+         * However, the `nimble-anchor` displays differently in color when the `href` attribute is absent.
+         * To ensure a consistent appearance, the font color is forced to the default link color regardless of the `href`
+         * attribute's presence.
+         *
+         * This visual consistency stems from the similarity between mention markdown and absolute links, with the distinction
+         * that mention markdown does not restrict the link scheme. When converting absolute links without HTTPS/HTTP to mention
+         * views, preserve the markdown in autolink format for proper rendering.
```

Comment:
Thanks for adding this comment. I might actually say it's *too* detailed, leaving it more likely to get out of date as the code evolves. Let's go with something more focused on the styling and less on the other details.
```suggestion
         *
         * See models/markdown-parser.ts where link elements are emitted for more info.
```

Reply:
Updated as per the suggestion.